variables.tf

```diff
@@ -353,6 +353,12 @@ DESCRIPTION
   nullable    = false
 }
 
+variable "private_link_excluded_zones" {
+  default = ["privatelink.pbidedicated.windows.net", "privatelink.analysis.windows.net", "privatelink.tip1.powerquery.microsoft.com"]
```

Comment:
Not a hill I'll die on but I would prefer if this worked out of the box for consumers, if someone wanted to use powerbi in this setup they just need to remove it from the exclusions list